### ISSUE TITLE
Add "Reconnect" Button to Hello Ethernaut Guide

### ISSUE THREAD

**Issue by johnlewissims** [OpenZeppelin/ethernaut#221]
I had meta mask installed before starting this project and was already on the Ropsten Network. When I connected to the site, console log threw a "Connected to Wrong Network" error. I switched networks in MetaMask, but am still getting the same error in console log. 

It would be nice to have a button to reconnect to the site on the first page and / or provide some directions in the "Connected to Wrong Network" error. 

I'm happy to submit a PR for this if the rest of the team is on board. Thank you! 😃

**Comment by frangio** [comment]
We're about to publish a big refactor of the frontend but a PR would be welcome after that.

**Comment by johnlewissims** [comment]
No problem! I'll wait for the refactor.
